### PR TITLE
Update structured logger

### PR DESCRIPTION
**Pull request by jaredpar**
Upgrade to latest structured logger binaries and see if that addresses the failure to read binary logs.

### CODE REVIEW

**Review by 333fred — APPROVED** [review]

**Review by RikkiGibson — APPROVED** [review]

File: eng/Directory.Packages.props [hunk 1]
```diff
@@ -319,7 +319,7 @@
       Infra
     -->
     <PackageVersion Include="Microsoft.Win32.Registry" Version="5.0.0" />
-    <PackageVersion Include="MSBuild.StructuredLogger" Version="2.2.169" />
+    <PackageVersion Include="MSBuild.StructuredLogger" Version="2.2.386" />
```
Comment (jjonescz) [inline eng/Directory.Packages.props:322]:
```suggestion
    <PackageVersion Include="MSBuild.StructuredLogger" Version="2.2.350" />
```

Comment (jaredpar) [reply eng/Directory.Packages.props:322]: Why that version vs. latest?

Comment (jjonescz) [reply eng/Directory.Packages.props:322]: Because of this error:
image

Comment (jjonescz) [reply eng/Directory.Packages.props:322]: It looks like rerun of the CI made the error go away. Interesting.

**Pull request merged**